tools/travis/travis_linux_before_install.sh

```diff
@@ -11,7 +40,7 @@ if ! [ -z ${CLANG_FORMAT+x} ]; then
     exit 0
 fi
 
-if [ -z ${DOCKER+x} ] && [ -z ${SONAR+x} ]; then
+if [ -z ${DOCKER+x} ] && [ -z ${SONAR+x} ] &&  [ -z ${ARCH} ]; then
```

Comment:
This change is not needed because you have an `exit 0` at the end of your block above.

Reply:
Fixed with https://github.com/open62541/open62541/pull/3023/commits/5f31676c03d8fe957e07f31f0cd336b460b4cec9